pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -199,6 +200,24 @@ message CommandConnected {
 	optional int32 protocol_version = 2 [default = 0];
 }
 
+message CommandAuthResponse {
+	required string client_version = 1;
+	required AuthData response = 2;
```

Comment:
make response and challenge members optional. protobuf generally recommends against making any fields required.